pkg/reconciler/inmemorychannel/dispatcher/inmemorychannel.go

```diff
@@ -209,11 +209,31 @@ func (r *Reconciler) patchSubscriberStatus(ctx context.Context, imc *v1.InMemory
 }
 
 // newConfigForInMemoryChannel creates a new Config for a single inmemory channel.
-func newConfigForInMemoryChannel(imc *v1.InMemoryChannel) (*multichannelfanout.ChannelConfig, error) {
+func newConfigForInMemoryChannel(ctx context.Context, imc *v1.InMemoryChannel) (*multichannelfanout.ChannelConfig, error) {
+	featureFlags := feature.FromContext(ctx)
```

Comment:
I know this is still WIP, but we need to add the featurestore to the context before (e.g. in controller.go via the ConfigStore controller.Option). Check for example the [Subscription controller](https://github.com/knative/eventing/blob/7a645f87cc1ec798a8171ef8de16d936de2036db/pkg/reconciler/subscription/controller.go#L72)